Theme prefix e sabd4 replicatset

aurelienCastellarnau opened the pull request:
# Contient:

 - Mise en place d'une option replicat set pour mongo: -replicat defaut false, rIp defaut "" et rp defaut 27017
 - CRUD sur Theme + ajout des themes dans le model + indexation des themes et des tarifs dans elastic search

Commit:
feat(all)

 - Add option replicat set for mongo (bool + ip and port as flags)
 - Change indexation, prefix all by abd4-
 - Add Theme management, crud + indexation

Commit:
feat(misc)

Kumatetsu commented on API/database/mongo/mongo.go:
```diff
@@ -20,6 +20,10 @@ import (
 )
 
+const (
+	ReplicaSetName = "c74b5276378ed3bd70cba37a3ac45fea"
```
abd4 non?

Kumatetsu commented on API/database/mongo/mongo.go:
```diff
@@ -40,3 +44,20 @@ func GetMongo(serverAddr string) (*mgo.Session, error) {
+
+func GetMongoReplicatSet(hosts []string, database string) (*mgo.Session, error) {
+	for _, host := range hosts {
+		fmt.Printf("%s %s", utils.Use().GetStack(GetMongoReplicatSet), host)
```
tiens je viens d'y penser, mais on est d'accord que ça ça te fais une sortie sur le terminal directement. Mais en soit en prod, tu feras tourner ton api en arrière plan donc c'est pas très utile. Ça serait pas mieux de créer des logs directs dans ces cas là?

Kumatetsu commented on API/handler/theme.go:
```diff
@@ -0,0 +1,173 @@
+}
+
+// Remove a theme by his description
+func RemoveThemeByTheme(ctx *context.AppContext, w http.ResponseWriter, r *http.Request) {
```
`RemoveThemeByTheme` c'est pas très explicit :D

Commit:
fix(review): El Juan

 - rename GetThemeByTheme => GetThemeByName
 - remove log in database package, must only return error and log higher
 - readd tests..
